### DISCUSSION TITLE
How do I view the sidebar on mobile?

### DISCUSSION THREAD

**Discussion by baduhai** [Xyphyn/photon#446] [General]
Photon works pretty well as a mobile client, I just haven't figured out how to view the sidebar on mobile. If there is no sidebar on mobile, how do I view my subscription list?

![Screenshot_20240916-225051](https://github.com/user-attachments/assets/4a3911d9-6864-4f83-bbc7-b825d374083b)

Where is it?

**Comment by sevonj** [comment]
It's kind of hard to find. Long press the photon icon at the corner.
![image](https://github.com/user-attachments/assets/fd788f1f-0290-4513-a434-a6dac81b7ab1)

**Reply by baduhai** [reply]
Thank you! I don't think I would have ever found it.

**Reply by sevonj** [reply]
Oh yeah, you're definitely not the only one

**Comment by baduhai** [comment]
@Xyphyn, I have a suggestion, perhaps swiping up from the dock/panel could reveal the command palette? When it's in panel mode, it would take the full width of the screen, whereas in dock mode, it would take the width of the dock, and also float. Of course, when the panel/dock is placed above, you'd swipe down instead. You can even keep the long pressing of the icon to reveal the command palette.

I've made a mockup of what I mean:
![mockup](https://github.com/user-attachments/assets/906601eb-b6f7-4c9c-bf7f-45b0c1e74b9c)

**Reply by Xyphyn** [reply]
I've experimented with that but it heavily conflicts with mobile navigation gestures, and is awful to do on desktop.

Btw the command palette is not the sidebar, but it's a way to access links from the sidebar.

**Reply by baduhai** [reply]
I see. It's just that the command palette was awfully difficult to find on mobile. Perhaps tapping on the empty space of the dock/panel could reveal the command palette?

Also, I wish the command palette was taller on mobile, so I could see more options at once.

> Btw the command palette is not the sidebar, but it's a way to access links from the sidebar.

Is there a way to access the sidebar on mobile?

**Reply by sevonj** [reply]
Yeah, I use gesture navigation and overlapping that in photon would be bad.

I believe that some confusion could be avoided if the sidebar was accessible, though.

Edit:
> Is there a way to access the sidebar on mobile?

No.

**Reply by baduhai** [reply]
> Yeah, I use gesture navigation and overlapping that in photon would be bad.

Same, which is why I suggested tapping on the empty section of the dock/panel.

**Reply by sevonj** [reply]
Sometimes there's no free space whatsoever. The dock has been adjusted before because it overflowed on my phone.
